### PR TITLE
Add thor command to start shotgun server.

### PR DESCRIPTION
**Pull request by cpursley**
Add `napa server` command to start the shotgun server similar to `rails server` in Ruby on Rails.

### CODE REVIEW

File: lib/napa/cli.rb [hunk 1]
```diff
@@ -63,6 +63,28 @@ def console(environment = nil)
         interpreter.start
       end
 
+      desc 'server', "Start the Napa server"
+      options aliases: 'c'
```
Comment (shaqq) [inline lib/napa/cli.rb]: should the alias be 's'?

Comment (cpursley) [inline lib/napa/cli.rb]: Indeed, thanks @shaqq

**Comment by cpursley** [comment]
Any recommendations on how to write tests for cli / thor code?

**Comment by umtrey** [comment]
+1 from me

**Comment by shaqq** [comment]
:+1: working as expected.

Maybe in the future we can have Napa configured to use whatever Rack server you want (unicorn, thin, mongrel, etc.), and `napa server` responds accordingly.

Nice add, @cpursley!

**Comment by cpursley** [comment]
I was thinking that exact same thing @shaqq regarding choice of Rack server.

**Comment by shaqq** [comment]
We also need to figure out a way to test these features, since many of the CLI commands don't work unless you're in the root directory of an app.

**Pull request merged**